experimental/framework/tree-react-api/api-report/tree-react-api.api.md

```diff
@@ -0,0 +1,58 @@
+## API Report File for "@fluid-experimental/tree-react-api"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { DataObject } from '@fluidframework/aqueduct/internal';
+import type { DataObjectClass } from '@fluidframework/fluid-static';
+import type { IFluidDataStoreFactory } from '@fluidframework/runtime-definitions/internal';
+import type { IFluidLoadable } from '@fluidframework/core-interfaces';
+import { ImplicitFieldSchema } from '@fluidframework/tree/internal';
+import * as React_2 from 'react';
+import { SchemaIncompatible } from '@fluidframework/tree/internal';
+import { TreeConfiguration } from '@fluidframework/tree/internal';
+import { TreeFieldFromImplicitField } from '@fluidframework/tree/internal';
+import { TreeNode } from '@fluidframework/tree';
+import { TreeView } from '@fluidframework/tree/internal';
+
+// @public
+export interface IReactTreeDataObject<TSchema extends ImplicitFieldSchema> extends ITreeDataObject<TSchema> {
+    readonly TreeViewComponent: (props: TreeViewProps<TSchema>) => React_2.JSX.Element;
+}
+
+// @public
+export interface ITreeDataObject<TSchema extends ImplicitFieldSchema> {
+    readonly config: TreeConfiguration<TSchema>;
+    readonly key: string;
+    readonly tree: TreeView<TSchema>;
+}
+
+// @public
+export interface SchemaIncompatibleProps {
+    readonly error: SchemaIncompatible;
+    readonly upgradeSchema: () => void;
+}
+
+// @public
+export function treeDataObject<TSchema extends ImplicitFieldSchema>(key: string, treeConfiguration: TreeConfiguration<TSchema>): DataObjectClass<IReactTreeDataObject<TSchema> & IFluidLoadable>;
+
+// @internal
+export function treeDataObjectInternal<TSchema extends ImplicitFieldSchema>(key: string, treeConfiguration: TreeConfiguration<TSchema>): DataObjectClass<IReactTreeDataObject<TSchema> & IFluidLoadable & DataObject> & {
+    readonly factory: IFluidDataStoreFactory;
+};
+
+// @public
+export interface TreeViewProps<TSchema extends ImplicitFieldSchema> {
+    readonly errorComponent?: React_2.FC<SchemaIncompatibleProps>;
+    readonly viewComponent: React_2.FC<{
+        root: TreeFieldFromImplicitField<TSchema>;
+    }>;
+}
+
+// @public
+export function useTree(subtreeRoot: TreeNode): void;
+
+// (No @packageDocumentation comment for this package)
```

Comment:
We should probably add a `@packageDocumentation` comment in `index.ts` so there is a package summary in the API docs.

Reply:
would be real slick if markdown magic could inject the package comment into the readme to avoid duplication.